panel/panel.html

```diff
@@ -90,6 +92,11 @@
 						},
 
 						todos: {
+							type: type.convert(class extends ObservableArray {
+								static get items() {
```

Comment:
Since this is just used in chrome, you can just do `static items = type.convert(ObservableObject);`. Both are fine, just wanted to make sure you knew this works also.

Reply:
I figured that was the case, but I'm currently doing the getter pattern in these cases expressly because eslint still chokes on static class properties.